Gemfile

```diff
@@ -10,6 +10,7 @@ gem "importmap-rails"
 gem "jbuilder", '~>2.12'
 gem 'jsonb_accessor', '~> 1.4'
 gem 'mailgun-ruby', '~>1.2.14'
+gem 'pagy'
```

Comment:
please pin this to the major version, i.e.:

`gem 'pagy', '~>9'`

Reply:
Done